start-tests.sh

```diff
@@ -7,97 +7,209 @@
+  echo -e "Detected clusters within the fleet: ${GREEN}${MANAGED_CLUSTERS[@]}${NC}\n"
+
+  if [[ -z $OPTIONS_MANAGED_BASEDOMAIN || -z $OPTIONS_MANAGED_USER || -z $OPTIONS_MANAGED_PASSWORD ]]; then
+    log_color "red" "One or more variables are undefined for imported cluster." "(set ${PURPLE}OPTIONS_MANAGED_BASEDOMAIN, OPTIONS_MANAGED_BASEDOMAIN, and OPTIONS_MANAGED_BASEDOMAIN${NC} to execute the test with environment variables)\n"
+
+    if [ ! -f $OPTIONS_MANAGED_KUBECONFIG ]; then
+      log_color "red" "The kubeconfig file for imported cluster was not located." "(set ${PURPLE}KUBECONFIG${NC} to ${YELLOW}${OPTIONS_MANAGED_KUBECONFIG}${NC} and oc login to create kubeconfig file."
+      echo -e "Skipping managed cluster test.\n"
+      export CYPRESS_SKIP_MANAGED_CLUSTER_TEST=true
+    else
+      # Exporting this variable so cypress will know to use the kubeconfig file for the imported cluster.
+      export CYPRESS_USE_MANAGED_KUBECONFIG=true
+      export CYPRESS_SKIP_MANAGED_CLUSTER_TEST=false
+
+      echo -e "Kubeconfig file detected at: $OPTIONS_MANAGED_KUBECONFIG - copying to ./cypress/config/import-kubeconfig"
+      cp $OPTIONS_MANAGED_KUBECONFIG ./cypress/config/import-kubeconfig
+      export OPTIONS_MANAGED_KUBECONFIG=./cypress/config/import-kubeconfig
+
+      if [[ -z $OPTIONS_MANAGED_KUBECONTEXT || "$OPTIONS_MANAGED_KUBECONTEXT" == "null" ]]; then
+        MANAGED_CLUSTER=($(oc config get-clusters --kubeconfig=$OPTIONS_MANAGED_KUBECONFIG))
+        export OPTIONS_MANAGED_KUBECONTEXT=default/${MANAGED_CLUSTER[1]}/kube:admin
+      fi
+
+      echo -e
+      log_color "cyan" "Switching context to log into Kube API server"
+      oc config use-context --kubeconfig=$OPTIONS_MANAGED_KUBECONFIG $OPTIONS_MANAGED_KUBECONTEXT
+      
+      export OPTIONS_HUB_BASEDOMAIN=$(oc whoami --show-server=true | cut -d'.' -f2- | cut -d':' -f1)
+
+      if [[ $OPTIONS_HUB_OC_IDP == "kube:admin" && -z $OPTIONS_HUB_USER ]]; then
+        export OPTIONS_HUB_USER=kubeadmin
+      fi
+
+      export OPTIONS_MANAGED_BASEDOMAIN=$(echo ${MANAGED_CLUSTER[1]} | cut -d'-' -f2- | cut -d':' -f1)
+
+      log_color "purple" "IMPORTED CLUSTER:" "$OPTIONS_MANAGED_BASEDOMAIN\n"
+    fi
+  else
+    echo -e "Environment variables detected. Configuring tests to execute with imported cluster exported variables."
+  fi
 fi
 
-log_color "cyan" "Logging into Kube API server"
-oc login --server=https://api.${CYPRESS_OPTIONS_HUB_BASEDOMAIN}:6443 -u $CYPRESS_OPTIONS_HUB_USER -p $CYPRESS_OPTIONS_HUB_PASSWORD --insecure-skip-tls-verify
+export CYPRESS_OPTIONS_MANAGED_BASEDOMAIN=$OPTIONS_MANAGED_BASEDOMAIN
+export CYPRESS_OPTIONS_MANAGED_CLUSTER_NAME=$OPTIONS_MANAGED_CLUSTER_NAME
+export CYPRESS_OPTIONS_MANAGED_KUBECONFIG=$OPTIONS_MANAGED_KUBECONFIG
+export CYPRESS_OPTIONS_MANAGED_KUBECONTEXT=$OPTIONS_MANAGED_KUBECONTEXT
+export CYPRESS_OPTIONS_MANAGED_PASSWORD=$OPTIONS_MANAGED_PASSWORD
+export CYPRESS_OPTIONS_MANAGED_USER=kubeadmin
```

Comment:
Export all of the CYPRESS_OPTIONS_MANAGED_* variables here. This will just make sure we're capturing the right value regardless of the testing process (i.e w/options.yaml, kubeconfig, or just exporting the variables).